docs/overview/introduction.md

```diff
@@ -40,6 +40,8 @@ Check out our guides to see how you can use Quickwit:
 - OTEL-native for [logs](../log-management/overview.md) and [traces](../distributed-tracing/overview.md)
 - [Jaeger-native](../distributed-tracing/plug-quickwit-to-jaeger.md)
 - [RESTful API](../reference/rest-api.md)
+- [Elasticsearch API](../reference/es_compatible_api.md) with ingest and query DSL support.
```

Comment:
Can you take the same features as the ones that will be on the readme page? 

see https://github.com/quickwit-oss/quickwit/pull/3491/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R47-R56

Reply:
Ok